libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Directory.Build.props

```diff
@@ -0,0 +1,28 @@
+<!-- Contains common properties that apply to projects under the Libraries folder -->
```

Comment:
The framework has its own Directory.Build.props that should apply here, can we delete this one so the other one takes effect. I can show you how to create one for this project if you need special settings.